lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -53,7 +53,7 @@ const expectations = [
                     'height': '>0',
                   },
                   'selector': '#aria-hidden-focus',
-                  'snippet': '<div id="aria-hidden-focus" aria-hidden="true">\n        <button>Focusable Button</button>\n      </div>',
+                  'snippet': '<div id="aria-hidden-focus" aria-hidden="true">',
```

Comment:
at first blush we're losing 'Focusable Button' but we actually have that in `nodeLabel`. so we good.

same with a few other of these cases.